sv_pipeline/exome/load_data.py

```diff
@@ -346,7 +346,7 @@ def get_es_schema(all_fields, nested_fields):
     return schema
 
 
-def export_to_elasticsearch(es_host, es_port, rows, index_name, meta, es_password, num_shards=6):
+def export_to_elasticsearch(es_host, es_port, rows, index_name, meta, es_password, num_shards=1):
```

Comment:
completely unrelated to the num_shards argument here, I just noticed this is `export_to_elasticsearch` in the exomes SV but `export_to_es` in the genomes (might be nice to rename one for consistency)

Reply:
given that this is an outage-related urgent fix I'm going to consider style fixes to be out of the scope of this PR